frontend/src/main/scala/bloop/exec/ForkProcess.scala

```diff
@@ -68,6 +69,9 @@ final case class ForkProcess(javaEnv: JavaEnv, classpath: Array[AbsolutePath]) {
     } else {
       val processBuilder = new ProcessBuilder(cmd: _*)
       processBuilder.directory(cwd.toFile)
+      val processEnv = processBuilder.environment()
+      processEnv.clear()
```

Comment:
Are we sure that we want to discard everything?

Reply:
Yeah, I elaborated on my other comment.